x-pack/plugins/data_usage/common/rest_types/usage_metrics.ts

```diff
@@ -62,22 +62,14 @@ export const UsageMetricsRequestSchema = {
         },
       }),
     ]),
-    dataStreams: schema.maybe(
-      schema.oneOf([
-        schema.arrayOf(schema.string(), {
-          minSize: 1,
-          validate: (values) => {
-            if (values.map((v) => v.trim()).some((v) => !v.length)) {
-              return '[dataStreams] list can not contain empty values';
-            }
-          },
-        }),
-        schema.string({
-          validate: (v) =>
-            v.trim().length ? undefined : '[dataStreams] must have at least one value',
-        }),
-      ])
-    ),
+    dataStreams: schema.arrayOf(schema.string(), {
```

Comment:
This schema has to be a oneOf(string, array). It should work with `api/path?dataStreams=xyz` and with `api/path?dataStreams=xyz&dataStreams=pqr`. The first one is a string and the second one is an array. So you can use the previous schema def without the `maybe` wrapper.

Reply:
Right.  I think in my head I was thinking this should be a POST request and we could simplify it and metricTypes to be arrays only.  Given they can pass in as many data streams as they want I think we should probably switch to a POST as I can see this URL getting very large. wdyt?